rtil/src/threads/ue/rebo/livesplit.rs

```diff
@@ -97,4 +126,42 @@ impl Run {
     pub fn attempt_count() -> u32 {
         LIVESPLIT_STATE.lock().unwrap().timer.run().clone().attempt_count()
     }
+    pub fn save_splits(user_input: &str) -> Result<(), SplitsSaveError> {
+        let state = LIVESPLIT_STATE.lock().unwrap();
+        let valid_splits_paths = VALID_SPLITS_PATHS.lock().unwrap();
+        let path = Path::new(user_input);
+        let (path, path_display) = match (path.is_relative(), valid_splits_paths.contains(path)) {
+            (true, _) => {
+                let sanitized = sanitize(user_input);
```

Comment:
Use `sanitize_filename::sanitize`. That way, readers know where a function is coming from. Usually, when there is no module or type prefix to a function, it's assumed to refer to a function in the same file.